### PR TITLE
fix: removed unnecessary endpoints

### PR DESCRIPTION
**Pull request by prahaladhchandrahasan**

### CODE REVIEW

**Comment by prahaladhchandrahasan** [comment]
I have removed unnecessary endpoints from `Loggers` and `Health` tab. The github workflow is failing for `Trace` tab

File: plugins/spring-boot/spring-boot.service.spec.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe("SpringBootService", () => {
 
     it("should return zero tabs", () => {
       // given
-      workspace.treeContainsDomainAndProperties.and.returnValues(false, false, false, false, false, false,false, false);
+      workspace.treeContainsDomainAndProperties.and.returnValues(false, false, false, false,false);
```
Comment (tadayosi) [inline plugins/spring-boot/spring-boot.service.spec.ts:66]: insert space: `false,false` -> `false, false`

File: plugins/spring-boot/health/health.service.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace SpringBoot {
 
     getHealth(): ng.IPromise<Health> {
       log.debug('Fetch health data');
-      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['healthEndpoint', 'Health'], ['getData', 'health'])
+      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['Health'], ['getData', 'health'])
```
Comment (tadayosi) [inline plugins/spring-boot/health/health.service.ts:14]: you should also remove `getData`

File: plugins/spring-boot/loggers/loggers.service.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ namespace SpringBoot {
     }
 
     getLoggerConfiguration(): ng.IPromise<LoggerConfiguration> {
-      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['loggersEndpoint', 'Loggers'], ['getLoggers', 'loggers'])
+      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['Loggers'], ['getLoggers', 'loggers'])
```
Comment (tadayosi) [inline plugins/spring-boot/loggers/loggers.service.ts:13]: you should also remove `getLoggers`

File: plugins/spring-boot/loggers/loggers.service.ts [hunk 4]
```diff
@@ -34,7 +34,7 @@ namespace SpringBoot {
     }
 
     setLoggerLevel(logger: Logger): ng.IPromise<void> {
-      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['loggersEndpoint', 'Loggers'], ['setLogLevel', 'configureLogLevel'])
+      const mbean: EndpointMBean = this.springBootService.getEndpointMBean(['Loggers'], ['setLogLevel', 'configureLogLevel'])
```
Comment (tadayosi) [inline plugins/spring-boot/loggers/loggers.service.ts:37]: you should also remove `setLogLevel`

**Review by tadayosi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**